src/components/Dropdown/Dropdown.tsx

```diff
@@ -253,6 +265,10 @@ export default class Dropdown extends AutoControlledComponent<
           selectedItem={search && !multiple ? undefined : null}
           getA11yStatusMessage={getA11yStatusMessage}
           onStateChange={changes => {
+            if (changes.isOpen !== undefined) {
+              this.trySetState({ open: changes.isOpen })
```

Comment:
When adding support for controlled mode, shouldn't we also add `onOpenChange` callback similar to what we have in popup?

Reply:
I am not adding controlled mode anymore because it's not as straightforward as expected, it was causing a regression and the issue might be on Downshift side.

Anyway, it was not needed anyway for this task, I just needed `isOpen` in styles.